### ISSUE TITLE
[Feature Request] Normalize audio

### ISSUE THREAD

**Issue by reyqn** [markokajzer/discord-soundbot#86]
We add lots of sounds with my friends, and not all have the same volume. This leads to some sounds being too loud while others are not well audible.

Would it be possible to play files at different volumes based on audio wave amplitude? Or maybe just have an option to normalize sounds when they are added?

**Comment by markokajzer** [comment]
We had some nice discussion around this topic on the support Discord. We came to the conclusion that the best way to handle this would be to introduce a new command that handles some basic sound manipulation. I will see what I can do.

**Comment by markokajzer** [comment]
Heyya, found motivation to add new commands. This should be fixed with a17ae67c407d7547d2d191e9af63393e212e4a6a.

You can now do something like `!modify <sound> volume <value>`

```
# Quadruples the volume of johncena
!modify johncena volume 4

# Halves the volume of airhorn
!modify airhorn volume 0.5
```

Feel free to try it out and let me know if you spot any issues with this new command.

---

Additionally you can also clip sounds with the same command via `!modify <sound> clip <start> <end>` Both start and end are specified as hh:mm:ss.xxx, where hours, minutes and milliseconds are optional.

```
# Clips the sound johncena from second four to second six
!modify johncena clip 4 6 

# Clips the sound airhorn with exact start and end time
!modify airhorn clip 1:23:22 1:23:42.111
```
